tests/common/mod.rs

```diff
@@ -52,9 +54,9 @@ pub const RAW_TRANSFER_GAS_LIMIT: u64 = 21_000;
 // TODO: Put somewhere better?
 pub fn for_each_block_from_disk(mut handler: impl FnMut(Block, InMemoryStorage)) {
     // Parse bytecodes
-    let bytecodes: Bytecodes = bincode::deserialize_from(BufReader::new(
-        File::open("data/bytecodes.bincode").unwrap(),
-    ))
+    let bytecodes: Bytecodes = bincode::deserialize_from(bufread::GzDecoder::new(BufReader::new(
```

Comment:
Can you add a `// TODO: Deduplicate logic with [bin/fetch.rs] when there is more usage`?

Reply:
Do you mean we should create a new method for decode logic? 
Is it necessary? 
Because they are in different modules, and the decode logic is only used in tests and the fetch bin. If extracting the decode logic, then put it in "src" folder?

Reply:
We don't need to do it with the current scope hence it being a `TODO`. The fear is once it's used in more places, we may update the path or data format at the write site but forget to update all read sites, etc.

Reply:
OK, Got it!